posts/2022-02-08-Rust-Survey-2021.md

```diff
@@ -0,0 +1,65 @@
+
+## A Global Community
+
+The Rust community continues to grow, with this survey having the largest number of complete survey responses (9354 respondents), exceeding last year's total by roughly 1500 responses.
+
+90% of respondents said that they use Rust for any purpose including learning (an all-time high!) with 5% stating they have in the past and 4% stating they have yet to use Rust.
+
+The survey was offered in 10 languages with 78% filling out the survey in English followed by Chinese (6%), German (4%), and French (3%). Despite English being the language most respondents completed the survey in, respondents hailed from all around the world. The United States was the country with the largest representation at 24% followed by Germany (12%), China (7%), and the U.K. (6%). In total 113 different countries and territories were represented through this survey!
+
+English, however, is not the language of choice for all Rustaceans with nearly 7% preferring not to use English as a language for technical communication. An additional 23% of respondents prefer another language in addition to English. The most common prefered languages (besides English) follow where Rustaceans live: Chinese, German, and French being the top 3. Japanese, Chinese, and Russian speakers were the most likely to prefer not to use English for technical conversation.
+
+## Using Rust
+
+The percentage of people using Rust continues to rise, with 81% of those currently using Rust using it on at least a weekly basis compared to 72% from last year's survey.
+
+75% of all Rust users say they can write production ready code, though 27% say that it is at times a struggle.
+
+![Rust expertise](/images/2022-02-RustSurvey/expertise.png)
+
+Overall, Rustaceans seem to be having a great time writing Rust with only 1% saying isn't fun to use. Only a quarter of a percent find Rust doesn't have any real benefit over other programming languages.
+
+## Rust at Work
+
+Rust can now safely be classified as a language used by people in professional settings. 59% of respondents who currently use Rust, use it at least occasionally at work with 23% using Rust for the majority of their coding. This is a large increase over last year where 58% of respondents didn't use Rust at work compared with this year's 41%.
+
+![Do you use Rust at work?](/images/2022-02-RustSurvey/rust-at-work.png)
+
+As for why respondents are using Rust work, the top answer was that it allowed users "to build relatively correct and bug free software" with 96% of respondents agreeing with that statement. After correctness, performance (92%) was the next most popular choice. 89% of respondents agreed that they picked Rust at work because of Rust's much-discussed security properties. Perhaps surprisingly, 89% of respondents also chose Rust because their teams found it fun and enjoyable to program (though respondents did agree more strongly with their choice to pick Rust due to its security properties).
```

Comment:
```suggestion
As for why respondents are using Rust at work, the top answer was that it allowed users "to build relatively correct and bug free software" with 96% of respondents agreeing with that statement. After correctness, performance (92%) was the next most popular choice. 89% of respondents agreed that they picked Rust at work because of Rust's much-discussed security properties. Perhaps surprisingly, 89% of respondents also chose Rust because their teams found it fun and enjoyable to program (though respondents did agree more strongly with their choice to pick Rust due to its security properties).
```